tests/test_utils/system_tests_class.py

```diff
@@ -27,7 +27,7 @@
 from airflow.utils import db
 from airflow.utils.log.logging_mixin import LoggingMixin
 
-DEFAULT_DAG_FOLDER = "example_dags"
+DEFAULT_DAG_FOLDER = "/opt/airflow/airflow/example_dags"
```

Comment:
That will only make the system tests run in Breeze - but not from the IDE. Is this what was intended?